src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  @include fb-jcc-bbox();
+
+  flex-direction: column;
+  align-items: center;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    padding-bottom: 40px;
+  }
+
+  &__description {
+    height: 36px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin: 0;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    align-self: start;
+
+    width: 98px;
+    height: 14px;
+
+    font-size: 10px;
+    color: $card-color--text-light;
+
+    margin: 0;
+    padding-bottom: 16px;
+  }
+
+  &__reviews {
+    @include fb-jcsb-aic();
+
+    width: 100%;
+    padding-bottom: 24px;
+
+    &-text {
+      font-size: 10px;
+      line-height: 16px;
+
+      margin: 0;
+    }
+  }
+
+  &__price {
+    @include fb-jcsb-aic();
+
+    width: 100%;
+    height: 18px;
+    padding-bottom: 16px;
+
+    &-label {
+      line-height: 18px;
+      color: $card-color--text-light;
+      margin: 0;
+    }
+
+    &-price {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+
+      margin: 0;
+    }
```

Comment:
again, I don't think these are correct, becuase these are element's of card
so either don't nest them and they are &__price-label or nest them, but then &__label